pkgs/development/python-modules/querystring_parser/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv, buildPythonPackage, fetchPypi, python
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "querystring_parser";
+  version = "1.2.4";
```

```suggestion
  version = "1.2.4";
  disabled = isPy27;
```
This seems to fail on python2, seeing as it's EOL, I would just disable it. No one should be using python2

```
running install tests
option -k not recognized
Usage: tests.py [options] [test] [...]

Options:
  -h, --help       Show this message
  -v, --verbose    Verbose output
  -q, --quiet      Minimal output
  -f, --failfast   Stop on first failure
  -c, --catch      Catch control-C and display results
  -b, --buffer     Buffer stdout and stderr during test runs

Examples:
  tests.py                               - run default set of tests
  tests.py MyTestSuite                   - run suite 'MyTestSuite'
  tests.py MyTestCase.testSomething      - run MyTestCase.testSomething
  tests.py MyTestCase                    - run all 'test*' test methods
                                               in MyTestCase

builder for '/nix/store/dz256kdp9zc80wfn5f9dwp6wqs1kv7sx-python2.7-querystring_parser-1.2.4.drv' failed with exit code 2
```

---

done and pushed!

---

looks like you rebased on top of master, but querystring_parser still failing

---

@jonringer ah sorry about that, I added `disabled = isPy27;` to `pythonPackages.mlflow` but not querystring_parser. Just added there, too.